apps/aechannel/src/aesc_codec.erl

```diff
@@ -233,43 +233,53 @@ dec_ch_reest_ack(<< ChainHash:32/binary
 
 
 -type fnd_created_msg() :: #{ temporary_channel_id := chan_id()
+                            , block_hash           := binary()
                             , data                 := binary()}.
 
 -spec enc_fnd_created(fnd_created_msg()) -> binary().
 enc_fnd_created(#{ temporary_channel_id := ChanId
+                 , block_hash           := BlockHash
                  , data                 := Data }) ->
     Length = byte_size(Data),
     << ?ID_FND_CREATED:1 /unit:8
      , ChanId         :32/binary
+     , BlockHash      :32/binary
      , Length         :2 /unit:8
      , Data           :Length/bytes >>.
 
 -spec dec_fnd_created(binary()) -> fnd_created_msg().
 dec_fnd_created(<< ChanId:32/binary
+                 , BlockHash:32/binary
```

Comment:
Doc: https://github.com/aeternity/protocol/blob/56793c15061a1b316ccb7506cea1daf2daeb3bd2/channels/OFF-CHAIN.md#funding_created

Doc needs update.

Versioning for incompatibility?

Reply:
In other points too.

Reply:
Addressed in aeternity/protocol#343